include/glow/Graph/Graph.h

```diff
@@ -565,6 +565,12 @@ class Function final : public Named {
   Node *createWeightedSum(llvm::StringRef name, llvm::ArrayRef<NodeValue> data,
                           llvm::ArrayRef<NodeValue> weights);
 
+  /// Create a series of nodes for the Clip operator. It limits the given input 
+  /// within an interval specified by the `min` and `max` arguments. They default
```

Comment:
These default values are based on the Caffe2 operators. However this node creator is independent of Caffe2, i.e. it could be used by the ONNX loader or elsewhere. So I think either this comment should be removed, or the you should add the defaults for the function's arguments.